docs/rules/rename-default-import.md

```diff
@@ -0,0 +1,61 @@
+# import/rename-default-import
+
+This rule will enforce a specific binding name for a default package import. Only ES6 imports are processed.
+
+
+## Rule Details
+
+Given:
+
+```js
+// ./foo.js
+export default function () { return 'Foo' }
+```
+
+and
+
+```json
+// .eslintrc
+{
+  "rules": {
+    "import/rename-default-import": [
+      "warn", {
+        "prop-types": "PropTypes", // key: name of the module, value: desired binding for default import
+        "./foo": "Foo"
```

Comment:
“./foo” is a relative path - meaning that it only makes sense inside a specific module. This could map to 10 different foo.js paths, for example. Is the intent to restrict all of them to the same name?

Reply:
what about just removing the docs/tests for this case? in the end we just look at it as a string - so it would still work if someone used it, but it'd simply be undocumented (it indeed doesn't make much sense to set this for something relative)

Reply:
If it doesn’t make sense, it shouldn’t be allowed - docs/tests are necessary